evalml/automl/automl_search.py

```diff
@@ -267,9 +270,9 @@ def __init__(self,
         # make everything ww objects
         self.X_train = _convert_to_woodwork_structure(X_train)
         self.y_train = _convert_to_woodwork_structure(y_train)
-
+        self.sampler = sampler if (sampler is not None and sampler != 'None') else None
```

Comment:
Oops sorry, this comment is out of date

Reply:
@dsherry I added the `sampler` argument so that users can pass in a string if they wanted to use a sampled `data_splitter`. I figured it would be a little easier for them to switch between different sampling methods compared to having to import them, but that was one of the questions I raised in the description of this PR. Do you think it'd make sense to only accept the `data_splitter` arg and remove `sampler`?